sous_chef/sous_chef/messaging/todoist_api.py

```diff
@@ -30,6 +30,19 @@ def __str__(self):
         return f"{self.message}: tag={self.tag} for value={self.value}"
 
 
+@dataclass
```

Comment:
As mentioned by developer, this should be taken out for a future commit.

Reply:
Removed for now